third_party/terraform/resources/resource_bigtable_gc_policy.go

```diff
@@ -112,7 +111,7 @@ func resourceBigtableGCPolicyCreate(d *schema.ResourceData, meta interface{}) er
 	}
 
 	instanceName := GetResourceNameFromSelfLink(d.Get("instance_name").(string))
-	c, err := config.bigtableClientFactory.NewAdminClient(project, instanceName)
+	c, err := config.BigQueryClientFactory(userAgent).NewAdminClient(project, instanceName)
```

Comment:
drive-by: looks like bigquery and bigtable got mixed up here! (and a handful of places below)

Reply:
https://github.com/GoogleCloudPlatform/magic-modules/pull/4057 fixes this, good catch!